src/Sentry/Platforms/Android/SentrySdk.cs

```diff
@@ -185,11 +189,11 @@ private static void InitSentryAndroidSdk(SentryOptions options)
         }
         else
         {
-            SentryAndroid.Init(AppContext, configuration);
+            SentryAndroid.Init(AppContext);
```

Comment:
This change doesn't seem related to the goal of this PR. Or is it?

Why was `configuration` arg removed?

Reply:
mistake my bad